src/main/webapp/WEB-INF/index.html

```diff
@@ -12,8 +12,19 @@
 <div class="container">
   <div class="tasks-container">
 
+    <div class="banner" id="info">
+      <p class="banner-text"> Hey! Select some tasks you would like to move to your calendar, and hit schedule! You can also select the timerange of scheduling, and set duration for tasks.</p>
+      <div class="mdc-touch-target-wrapper banner-button" onclick="hideInfo()">
+        <button class="mdc-button mdc-button--touch">
+          <div class="mdc-button__ripple"></div>
+          <span class="mdc-button__label">GOT IT</span>
```

Comment:
Just use normal casing in HTML and if you want the style to be all uppercase, instead use the CSS property https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform